docs/content/getting-started/kubernetes.md

```diff
@@ -0,0 +1,386 @@
+---
+title: "Kubernetes and Traefik Quick Start"
+description: "Deploy Traefik in Kubernetes using Helm and expose your first service"
+---
+
+# Getting Started with Kubernetes and Traefik
+
+Kubernetes is a first-class citizen in Traefik, offering native support for Kubernetes resources and the latest Kubernetes standards. 
+Whether you're using Traefik's [IngressRoute CRD](../reference/routing-configuration/kubernetes/crd/http/ingressroute.md) or the Kubernetes Gateway API, 
+Traefik provides a seamless experience for managing your Kubernetes traffic.
+
+This guide shows you how to:
+
+- Create a Kubernetes cluster using kind or k3d
+- Install Traefik using Helm
+- Expose the Traefik dashboard
+- Deploy a sample application
+- Configure basic routing with IngressRoute and Gateway API
+
+## Prerequisites
+
+- Kubernetes
+- Helm 3
+- kubectl
+- kind or k3d (for local cluster creation)
+
+## Create a Kubernetes Cluster
+
+### Using kind
+
+kind requires specific configuration to use an IngressController on localhost. Create a configuration file:
+
+```yaml
+# kind-config.yaml
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+name: traefik
+nodes:
+- role: control-plane
+  extraPortMappings:
+  - containerPort: 80
+    hostPort: 80
+    protocol: TCP
+  - containerPort: 443
+    hostPort: 443
+    protocol: TCP
+```
+
+Create the cluster:
+
+```bash
+kind create cluster --config=kind-config.yaml
+```
+
+Configure kubectl and wait for the node to be ready:
+
+```bash
+kubectl cluster-info
+kubectl wait --for=condition=ready nodes traefik-control-plane
+```
+
+Add a load balancer like [MetalLB](https://metallb.io/) or [cloud-provider-kind](https://github.com/kubernetes-sigs/cloud-provider-kind) to it:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/v0.13.11/config/manifests/metallb-native.yaml
+kubectl wait --namespace metallb-system --for=condition=ready pod --selector=app=metallb --timeout=90s
+```
+
+!!! note
+    If using MetalLB, ensure you assign an IP address pool for the LoadBalancer and advertise it.
+
+### Using k3d
```

Comment:
Since k3d is easier to set up (LB already there), wdyt about putting it before _using kind_ section ?